sys/dev/isp/isp_freebsd.c

```diff
@@ -986,6 +986,16 @@ isp_target_start_ctio(ispsoftc_t *isp, union ccb *ccb, enum Start_Ctio_How how)
 			continue;
 		}
 
+		/*
+		 * Is this command a dead duck?
+		 */
+		if (atp->dead) {
+			isp_prt(isp, ISP_LOGERR, "%s: [0x%x] not sending a CTIO for a dead command", __func__, cso->tag_id);
```

Comment:
I see we are calling `isp_dump_atpd` above, just wanted to make if we need to dump `atpd` here as well.

Reply:
This chunk I practically restored as it was before my previous commit.  But no, I don't think `isp_dump_atpd()` would be helpful here, since it dumps all the ATIOs, which we don't need here.  We already have one we need, it is just dead, that we already log.